INSTALL.md

```diff
@@ -1,9 +1,9 @@
 Manual Installation
 -------------------
 
-### Dependencies
+This guide is compatible with Ubuntu 18.04 and Ubuntu 20.04
```

Comment:
In fact, we are using Debian on our infrastructure, but we can keep Ubuntu.

By the way, how do you know if it is compatible with Ubuntu 20.04, as it wasn't yet released?

Reply:
Ubuntu 20.04 beta was released, I followed the procedure on an [ubuntu:20.04](https://github.com/tianon/docker-brew-ubuntu-core/blob/857debd4347496c541f9806c084710dace49cd27/focal/Dockerfile) docker image.

I can also try on recent debian versions if necessary.